### ISSUE TITLE
How to call Multiple function from Validator field??

### ISSUE THREAD

**Issue by Shraddha80011** [mercmobily/SimpleSchema#2]
I want to call more than one function in **validator** from complex schema, 
my code like this

    var fieldValidatorFunc =  function( obj, value, fieldName ){
     alert("fieldValidatorFunc")
    }; 
    var ABCValidatorFunc =  function( obj, value, fieldName ){
      alert("ABCValidatorFunc")
    };
    complexSchema = new Schema({
      age: {
        type: 'number',
        default: 15,
        min: 0,
        max: 130,
        validator: fieldValidatorFunc, ABCValidatorFunc, extraParameter: true 
      }
    });

But it is only called **fieldValidatorFunc**, it doesn't call **ABCValidatorFunc**.
I want to call both function at a time **OR** one by one.
Thanks.

**Comment by mercmobily** [comment]
Hi,

Are you sure you wrote the code correctly?
When you say:

    validator: fieldValidatorFunc, ABCValidatorFunc, extraParameter: true

It doesn't strike me as valid Javascript code, quite frankly.
Running it with Chrome, it ends up only assigning `fieldValidatorFunc` to `validator`.

Maybe a possible syntax would be:

    validator: [ fieldValidatorFunc, ABCValidatorFunc ]

Is that something you'd like to see?

I wrote this code to support [JsonRestStores](https://github.com/mercmobily/JsonRestStores). However, I will be happy to improve it.

Merc.

**Comment by Shraddha80011** [comment]
Yes, I know that my code is not correct, but I want to show you How to call two function in validator field??
And you suggest me a way to call functions but it is also not working it shows me error like this,
**Validator function needs to be a function, found: object**.
Then now please give me another way to call more than one function in Validator field.
Thanks

**Comment by mercmobily** [comment]
Sorry to ask you, but... would you be able to provide a real-life use case for this need?
In general, the best thing to do is to use one validator, which will then apply several checks. These checks can and should be implemented in other functions, so that you can "include" any check you like...
